### PR TITLE
Ripple protobuf messages

### PR DESCRIPTION
**Pull request by tsusanka**
Adds Ripple protobuf messages. Other Ripple PRs will follow in trezor-core and trezor-python after this is merged.

### CODE REVIEW

File: protob/messages-ripple.proto [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	optional string address = 1;	// Address in Ripple format (base58 of a pubkey with checksum)
+}
+
+/**
+ * Request: ask device to sign Ripple transaction
+ * @next RippleTxTypeRequest
+ */
+message RippleSignTx {
+	repeated uint32 address_n = 1;				// BIP-32 path. For compatibility with other wallets, must be m/44'/144'/index'
+	optional string account = 2;				// source account address. This address must be equal to the one derived from the BIP-32 path
```
Comment (prusnak) [inline protob/messages-ripple.proto]: Can we drop this? It seems redundant.

Comment (tsusanka) [reply protob/messages-ripple.proto]: I wasn't sure about that either. The reason I left it was that it is a confirmation that the user is sending funds from the account he assumes. It is used only [here](https://github.com/trezor/trezor-core/blob/tsusanka/ripple/src/apps/ripple/sign_tx.py#L18).

But I'm not keen on it, maybe it's a bit far-fetched.

**Comment by tsusanka** [comment]
@prusnak I think/hope this is the final version, so it's ready for review

File: protob/messages-ripple.proto [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	optional string address = 1;	// Address in Ripple format (base58 of a pubkey with checksum)
+}
+
+/**
+ * Request: ask device to sign Ripple transaction
+ * @next RippleSignedTx
+ */
+message RippleSignTx {
+	repeated uint32 address_n = 1;				// BIP-32 path. For compatibility with other wallets, must be m/44'/144'/index'
+	optional uint32 fee = 2;					// fee (in drops) for the transaction
```
[the remark this answers is not in the document]
Comment (tsusanka) [reply protob/messages-ripple.proto]: You're correct. Same goes for the payment's `amount`, I guess. I fixed both

**Comment by prusnak** [comment]
LGTM Merged

**Pull request merged**